index.js

```diff
@@ -56,16 +56,31 @@ class NpmAddDependencies {
 
   runNpmShow(dep) {
     return new Promise((resolve) => {
-      npmRun.exec(`npm show ${dep} dist-tags`, (err, stdout) => {
+      npmRun.exec(`npm show ${dep} dist dist-tags`, (err, stdout) => {
         if (!err) {
           const parsed = stdout.match(/latest: '(.*?)'/i);
 
           if (!parsed || undefined === parsed[1]) {
-            console.error(`Could not obtain the latest version for: ${dep}. Skip.`);
+            if (!dep.includes('@')) {
```

Comment:
If we do the thing from line 67's comment, here we can check for `undefined === depVersion`

Reply:
Please see [above comment](https://github.com/arfeo/npm-add-dependencies/pull/5#discussion_r314966075)